src/main/java/org/whispersystems/textsecuregcm/sms/TwilioSmsSender.java

```diff
@@ -67,8 +67,13 @@ public void deliverSmsVerification(String destination, String verificationCode)
     List<NameValuePair> messageParams  = new LinkedList<>();
     messageParams.add(new BasicNameValuePair("To", destination));
     messageParams.add(new BasicNameValuePair("From", number));
-    messageParams.add(new BasicNameValuePair("Body", SmsSender.SMS_VERIFICATION_TEXT + verificationCode));
-
+    	
+	if ("ios".equals(clientType)) {
+		messageParams.add(new BasicNameValuePair("Body", String.format(SmsSender.SMS_IOS_VERIFICATION_TEXT, verificationCode, verificationCode)));
+    } else {
+    	messageParams.add(new BasicNameValuePair("Body", String.format(SmsSender.SMS_VERIFICATION_TEXT, verificationCode)));
+    }
+	
```

Comment:
Looks like you accidentally added some tabs instead of spaces here :) (and in `NexmoSmsSender.java`)